src/chromium/interoperability-with-cpp.md

```diff
@@ -0,0 +1,33 @@
+# Interoperability with C++
+
+The Rust community offers multiple options for C++/Rust interop, with new tools
+being developed all the time. At the moment, Chromium uses a tool called "cxx".
+
+Behind the scenes, this uses a "lowest common denominator" C ABI, but it builds
+nice abstractions on both sides.
```

Comment:
This seems like too much details - most people won't care about C ABI and might not even know what that is.  

Option #1: Maybe remove this paragraph?
Option #2: Maybe replace with something that talks about specific, most-important benefits of using the tool:

1. The tool auto-generates declarations for functions and types defined in the other language (both ways)
2. The tool ensures that the build fails if the generated declarations don't match the definitions (e.g. because of mistakes in the `#[cxx::bridge(...)]` manifest)

Reply:
I agree, done.